yazi-fs/src/folder.rs

```diff
@@ -149,7 +149,7 @@ impl Folder {
 		self.offset = if self.cursor < (self.offset + limit).min(len).saturating_sub(scrolloff) {
 			len.saturating_sub(limit).min(self.offset)
 		} else {
-			len.saturating_sub(limit).min(self.cursor.saturating_sub(limit) + scrolloff)
+			len.saturating_sub(limit).min(self.cursor.saturating_sub(limit - 1) + scrolloff)
```

Comment:
limit could be 0, when the terminal window is too small, which causes the app to crash, please fix it

Reply:
okay, it should be fine now